lib/Views/src/layouts/DateTimeView64x64.cpp

```diff
@@ -286,6 +296,74 @@ void DateTimeView64x64::drawAnalogClockHand(YAGfx& gfx, int16_t minute, int16_t
         col);
 }
 
+
+void DateTimeView64x64::getConfiguration(JsonObject& jsonCfg) const
+{
+    JsonObject jsonAnalogClock = jsonCfg.createNestedObject("analogClock");
+
+    jsonAnalogClock["secondsMode"] = m_secondsMode;
+
+    for (uint32_t index = 0U;  index < ANA_CLK_COL_MAX; ++index)
+    {
+        jsonAnalogClock[ANALOG_CLOCK_COLOR_KEYS[index]]= Util::colorToHtml(m_analogColors[index]);
+    }
+}
+
+bool DateTimeView64x64::setConfiguration(const JsonObjectConst& jsonCfg)
+{
+    bool result = true;

```

Comment:
The result will be true even if "analogClock" would be a string instead an object.

Reply:
I don't understand this comment. 
Wouldn't RestApiTopicHandler::par2Json() have done this convertion bevore this call already ?

Reply:
If "analogClock" is Null, the function will return true. But shouldn't it return false in this case?

Reply:
Will change to fail on missing data. The old code was meant to be "backward compatible" and treated analog clock data as optional.  This makes no sense anymore as this is an analogClock supporting view class now. So the data must be there.